src/add/form-data.browser.js

```diff
@@ -9,6 +9,14 @@ exports.toFormData = async input => {
   let i = 0
 
   for await (const file of files) {
+    if (file.mtime) {
+      formData.append('mtime', file.mtime)
```

Comment:
Need to identify which file/dir this metadata pertains to.

Reply:
This has been refactored out.